### ISSUE TITLE
TypeError: null is not an object (evaluating 'getModal().querySelectorAll')

### ISSUE THREAD

**Issue by dirchev** [limonte/sweetalert2#523]
This error happened on Chrome Mobile iOS 57. The error was reported from one of the systems, that I take part in developing.

1. The problem was that there was [`handleKeyDown`](https://github.com/limonte/sweetalert2/blob/master/src/sweetalert2.js#L589) function triggered (I think it there might be a virtual keyboard left open when on the iPhone) and then [`getFocusableElements`](https://github.com/limonte/sweetalert2/blob/master/src/sweetalert2.js#L600)

2. Inside this function [`getModal`](https://github.com/limonte/sweetalert2/blob/32ec66e7c062c4b6cb44c76d1d8ac04c23347df4/src/utils/dom.js#L130) method is called, which uses `getContainer().querySelector(...)`.

This is where the error was thrown. The DOM element with class `swalClasses.container` was not found, therefore `null` is returned from the execution of the [`getContainer()`](https://github.com/limonte/sweetalert2/blob/32ec66e7c062c4b6cb44c76d1d8ac04c23347df4/src/utils/dom.js#L128) method and `getContainer().querySelector()` is invalid.

---

I can not provide steps to reproduce, because I was not able to reproduce the error either

---

I think the right solution here is to handle not found container element in the `getModal` method. All the methods that use it must be updated accordingly to accept `null` as a result of the execution of `getModal()`.

As I am not familiar with this lib, so I invite you to have a discussion before proceeding with PR :)

**Comment by brtinney-touchsuite** [comment]
I'm encountering the same problem in Chrome 58.0.3029.96 on macOS Sierra.

Able to reproduce it with this:

```javascript
swal("test");
swal("test2");
swal.close();
// Then press escape
```

I'm guessing the problem is that the overwritten modal doesn't have its events unbound.

**Comment by limonte** [comment]
Thanks guys for such a good bug report! I'll handle it asap.

**Comment by alencarandre** [comment]
It occurred also in chrome 58 on desktop when does not have catch in promise returned and have cancel button.

**Comment by limonte** [comment]
Fixed in [v6.6.3](https://github.com/limonte/sweetalert2/releases/tag/v6.6.3)

Thanks @dirchev and @brtinney-touchsuite for your help in debugging this issue!
